.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+
+/samples/                           @DanSanche @googleapis/python-samples-owners
```

Comment:
One thing Jeff mentioned doing for Java was using `/samples/**/*.py` so that renovate PRs don't need to trigger this. Do we want to do that here?

Reply:
SGTM, updated,